NetKAN/USAConversion.netkan

```diff
@@ -0,0 +1,27 @@
+{
+    "spec_version": 1,
+    "license": "CC-BY-NC-SA-4.0",
+    "$kref": "#/ckan/kerbalstuff/962",
+    "identifier": "USAConversion",
```

Comment:
Also, the identifier may make more sense as `NASACareerModeTotalConversion` or something, the only reference I can find to `USAConversion` is in the install location.